examples/kubernetes-kata-containers.json

```diff
@@ -0,0 +1,50 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.10",
+      "kubernetesConfig": {
+        "networkPlugin": "flannel",
+        "containerRuntime": "kata-containers",
+        "addons": [
```

Comment:
any reason why tiller and dashboard are disabled explicitly? If not, we can probably let them go to default

Reply:
No, this was just me being lazy and copying what was done for Clear Containers in the past.